pkg/plugin/predicates/predictor.go

```diff
@@ -201,6 +203,16 @@ func (p *Predictor) init() {
 			return predicates.NewVolumeBindingPredicate(args.VolumeBinder)
 		},
 	)
+
+	// GeneralPredicates are the predicates that are enforced by all Kubernetes components
+	// (e.g. kubelet and all schedulers)
+	p.RegisterFitPredicate(predicates.GeneralPred, predicates.GeneralPredicates)
+
+	// Fit is determined based on whether a pod can tolerate all of the node's taints
+	p.RegisterMandatoryFitPredicate(predicates.PodToleratesNodeTaintsPred, predicates.PodToleratesNodeTaints)
+
+	// Fit is determined based on whether a pod can tolerate unschedulable of node
+	p.RegisterMandatoryFitPredicate(predicates.CheckNodeUnschedulablePred, predicates.CheckNodeUnschedulablePredicate)
```

Comment:
Now Predictor#mandatoryFitPredicates is set but not used, should we support adding all mandatory-fit-predicates when initializing scheduler policy in predctor.go#NewPredictor ?

Reply:
Yes, I think we should register all predicates in the init, in order to fully support all existing predicates. If we have anything missing, we need another ticket to add them back.